external-crates/move/crates/move-compiler/tests/move_2024/typing/syntax_methods_args_polymorphic_invalid.exp

```diff
@@ -0,0 +1,11 @@
+error[E04034]: 'syntax' method types differ
+  ┌─ tests/move_2024/typing/syntax_methods_args_polymorphic_invalid.move:6:43
+  │
+6 │     public fun borrow<T,Q>(_s: &S<T>, _j: Q): &T { abort 0 }
+  │                                           ^ This index function expects type 'Q'
+  ·
+9 │     public fun borrow_mut<T,Q>(_s: &mut S<T>, _j: T): &mut T { abort 0 }
+  │                                                   - This mutable index function expects type 'T'
+  │
```

Comment:
Curious, what does the error message look like if I do
```
public fun borrow<Q,T>(_s: &S<T>, _j: T): &Q { abort 0 }
public fun borrow_mut<T,Q>(_s: &mut S<T>, _j: T): &mut T { abort 0 }
``` 
I imagine it will do something like, expects 'T' and expects 'T'?